src/wmtk/TetMeshOperationExecutor.cpp

```diff
@@ -610,7 +562,7 @@ Tuple TetMesh::TetMeshOperationExecutor::split_edge()
             tf = tf_accessor.index_access().const_vector_attribute(t_old);
```

Comment:
You create the accessors twice. I think it would make sense to drag them out of the if-statement (line 484) and just create them once.